.vuepress/components/LevelWithButton.vue

```diff
@@ -3,13 +3,13 @@
     <!-- Left side -->
     <div class="level-left">
       <div v-if="!imageRight">
-        <figure class="image ">
+        <figure class="image">
           <img :src="image" />
         </figure>
       </div>
       <div v-else>
         <div class="is-size-10 paragraph">{{desc}}</div>
-        <a :href="link">
+        <a :href="link" class="mybutton">
```

Comment:
Follow css naming convention. This would be my-button

I would rename it to action-button